packages/plugin-webpack/src/plugin.test.ts

```diff
@@ -1,233 +1,261 @@
 // Allow Jest snapshots because the test outputs are illegible.
 /* eslint-disable jest/no-restricted-matchers */
 
-import webpack, { Stats } from 'webpack';
-import { createFsFromVolume, Volume } from 'memfs';
-import SnapsWebpackPlugin from './plugin';
+import webpack, { Stats, Configuration } from 'webpack';
+import { createFsFromVolume, IFs, Volume } from 'memfs';
+import { IPromisesAPI } from 'memfs/lib/promises';
+import { checkManifest, evalBundle } from '@metamask/snap-utils';
+import SnapsWebpackPlugin, { Options } from './plugin';
+import { DEFAULT_SNAP_BUNDLE, getSnapManifest } from './__test__';
+
+jest.setTimeout(10000);
```

Comment:
Do these tests take this long to run?

Reply:
Nope, I added this for some debugging and forgot to remove it.